docker-compose.yml

```diff
@@ -31,22 +34,29 @@ services:
     restart: always
     ports:
       - "3001:3000"
+    networks:
```

Comment:
Why did you add the network? This is not necessary

Reply:
we need all of the containers to run on the same network

Reply:
@JosephOri They already are..